kpi/models/asset_version.py

```diff
@@ -32,7 +32,7 @@ class AssetVersion(models.Model):
     version_content = JSONBField()
     uid_aliases = JSONBField(null=True)
     deployed_content = JSONBField(null=True)
-    _deployment_data = JSONBField(default=False)
+    _deployment_data = JSONBField(default=dict)
```

Comment:
The main reason why I changed it, it's because Django complains about something else than a dict as a default. 
```
kpi.AssetVersion._deployment_data: (postgres.E003) JSONField default should be a callable instead of an instance so that it's not shared between all field instances.
kpi_1             | 	HINT: Use a callable instead, e.g., use `dict` instead of `{}`.
```

Reply:
Django bug :wink: `False` ain't mutable
![i-do-not-support-this-man](https://user-images.githubusercontent.com/2085013/69075046-a69ef580-09fe-11ea-9fdd-b65d1c4d5fb2.gif)